extensions/amp-analytics/0.1/vendors.js

```diff
@@ -644,6 +644,76 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'linkpulse': {
+    'vars': {
+      'id': '',
+      'pageUrl': 'CANONICAL_URL',
+      'title': 'TITLE',
+      'section': '',
+      'channel': 'amp',
+      'type': '',
+      'host': 'pp.lp4.io',
```

Comment:
Yes, these are variables that publishers can changes as they want. Ie. Host variable can point different hosts (like testing, staging and even other custom servers if needed).